server/server.js

```diff
@@ -15,8 +14,7 @@ app.get('/sitemap.xml', function(req,res) {
   res.sendFile(path.join(__dirname, '../sitemap.xml'))
 });
 
-app.use('/d5', d5);
-app.use('/d4', d4);
+app.use('/plod',plodC);
```

Comment:
not so sure about this: you create a dependency on running the server again, right? This file should be (almost) empty if the server was just used to service the static html and compiled js.